### ISSUE TITLE
Failed to install a plugin; Failed to download tflint-ruleset-terraform_windows_amd64.zip: file not found in the GitHub release

### ISSUE THREAD

**Issue by chaoscreater** [terraform-linters/tflint#1777]
### Summary

`Failed to install a plugin; Failed to download tflint-ruleset-terraform_windows_amd64.zip: file not found in the GitHub release. `

The filename is different to the actual release:
https://github.com/terraform-linters/tflint-ruleset-azurerm/releases/download/v0.24.0/**tflint-ruleset-azurerm_windows_amd64.zip**

### Command

tflint /init

### Terraform Configuration

```terraform
HCL
```

### TFLint Configuration

```HCL
plugin "terraform" {
  enabled = true
  preset  = "recommended"
  version = "0.24.0"
  source  = "github.com/terraform-linters/tflint-ruleset-azurerm"
}
```

### Output

```Shell
Installing `terraform` plugin...
Failed to install a plugin; Failed to download tflint-ruleset-terraform_windows_amd64.zip: file not found in the GitHub release. Does the release contain the file with the correct name ?

```

### TFLint Version

0.46.1

### Operating System

- [ ] Linux
- [ ] macOS
- [X] Windows

**Comment by wata727** [comment]
The asset name is determined from the block's label. Match the source and the label name.

```diff
- plugin "terraform" {
+ plugin "azurerm" {
    enabled = true
    version = "0.24.0"
    source  = "github.com/terraform-linters/tflint-ruleset-azurerm"
  }
```
